Editor/Scripts/Utilities/MeshSyncObserver.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using UnityEditor;
+
+namespace Unity.MeshSync.Editor.Analytics {
```

Comment:
The namespace does not match the folder location. Might be better the file to be moved under Editor/Scripts/Analytics